wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -562,7 +562,7 @@ async def run(
             raise LaunchError(
                 f"Unexpected exception when creating Kubernetes job: {str(e)}\n"
             )
-        job_response = response[0][0]
+        job_response = response[0]
```

Comment:
this fixes a regression caused here https://github.com/wandb/wandb/pull/6730

We switched from `kubernetes` to `kubernetes_asyncio`, which eventually led to us switching from `create_from_yaml` to `create_from_dict` which has a different output shape. I didn't update the tests accordingly and now when the agent submits a vanilla k8s job we inevitably get this error:
```
TypeError: 'V1Job' object is not subscriptable
```